### ISSUE TITLE
Link Tag Issue

### ISSUE THREAD

**Issue by GeoTimber** [mapbox/togeojson#23]
Hi and thanks for GeoJSON
Link Tags in GPX files are not translating to properties.
```
source:
<link href="http://www.ayeltd.biz"><text>AYE Ltd.</text></link>
result:
feature link property = does not exist

source:
<link href="http://www.ayeltd.biz">
	<text>AYE Ltd.</text>
</link>
result:
feature link property "\n     " (a control char)
```
Not sure how they should translate,You can separate attributes and node values.
link.attribs.href=http://www.ayeltd.biz
link.text="AYE Ltd."
Or dont convert links.
Hope this is useful.

**Comment by joukewitteveen** [comment]
Also, there may be many links. What would be the preferred way to represent them? As a `links` array that contains objects `{ href: "...", text: "...", type: "..." }` objects for every link?

**Comment by tmcw** [comment]
Fixed in #109!
